content/docs/dvclive/api-reference/live/log_plot.md

```diff
@@ -0,0 +1,150 @@
+# Live.log_plot()
+
+```py
+def log_plot(self, name: str, labels, predictions, **kwargs):
+```
+
+#### Usage:
+
+```py
+from dvclive import Live
+
+live = Live()
+
+y_true = [0, 0, 1, 1]
+y_pred = [1, 0, 1, 0]
+y_score = [0.1, 0.4, 0.35, 0.8]]
+live.log_plot("roc", y_true, y_score)
+live.log_plot("confusion_matrix", y_true, y_pred)
+```
+
+## Description
+
+Generates a
+[scikit learn plot](https://scikit-learn.org/stable/visualizations.html) and
+saves the result in `{path}/plots/{name}.json`.
+
+💡 The generated `{path}/plots/{name}.json` can be visualized with `dvc plots`.
+
+### Step updates
+
+⚠️ `Live.log_plot()` can't be currently used in scripts where the step is being
+updated ( with `Live.next_step()` or `Live.set_step()`).
```

Comment:
I think we need to clarify step behavior for `log_image` and `log_plot`. It's possible to do something like:

```
for epoch in range(NUM_EPOCHS):
    live.log(metric_name, value)
    live.next_step()

live.set_step(None)
live.log_plot("roc", y_true, y_score)
live.log_image("custom_image", img)
```

This allows logging summary plots/images at the end of training.

So maybe it's more accurate to say that `log_image` and `log_plot` can only be used when the `step` value is None/undefined.

Do you think an example like this would be helpful in the docs?

Reply:
I included the example under the `step updates` section